packages/react-ui-core/src/DateInput/DateInput.js

```diff
@@ -1,24 +1,33 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
-import DatePicker from 'react-datepicker'
+import DatePicker from 'react-basic-datepicker'
 
 export default class DateInput extends Component {
+  static propTypes = {
+    selected: PropTypes.object,
+    onChange: PropTypes.func,
+    maxDate: PropTypes.object,
```

Comment:
The props `maxDate` and `minDate` don't seem to be modified at all in this component (no default value, not passed explicitly to `<DatePicker>` except through `{...rest}` like any other unknown props) so should they just be omitted from propTypes?

It's probably nice to document which props we expect to normally support for `DateInput` but that could be done with some comments.